src/include/lean/lean.h

```diff
@@ -40,10 +40,12 @@ extern "C" {
 #define LEAN_UNLIKELY(x) (__builtin_expect((x), 0))
 #define LEAN_LIKELY(x) (__builtin_expect((x), 1))
 #define LEAN_ALWAYS_INLINE __attribute__((always_inline))
+#define LEAN_NOINLINE __attribute__((noinline))
```

Comment:
I don't see a good reason for defining it here at `lean.h` (our public interface).

Reply:
Inline in `alloc.cpp` then, until we have a need for it in other files?